cmd/root/bundle_test.go

```diff
@@ -108,8 +118,9 @@ func TestBundleConfigureWithMismatchedProfile(t *testing.T) {
 	err := cmd.Flag("profile").Value.Set("PROFILE-1")
 	require.NoError(t, err)
 
-	err = setupWithHost(t, cmd, "https://x.com")
-	assert.ErrorContains(t, err, "the host in the profile (https://a.com) doesn’t match the host configured in the bundle (https://x.com)")
+	diags := setupWithHost(t, cmd, "https://x.com")
+	require.NotEmpty(t, diags)
+	assert.Contains(t, diags[0].Summary, "cannot resolve bundle auth configuration")
```

Comment:
The new assertion is not equivalent to the old one.

Reply:
it's the same. update the assert to include full message